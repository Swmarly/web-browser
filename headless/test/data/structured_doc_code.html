<!DOCTYPE html>
<html lang="en">
<body>
<code>
Content in html code.
</code>
</body>
</html>
