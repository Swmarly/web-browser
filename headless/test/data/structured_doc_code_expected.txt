{
   "lang": "en",
   "type": "Document",
   "~children": [ {
      "type": "Div",
      "~children": [ {
         "type": "Code",
         "~children": [ {
            "type": "NonStruct"
         } ]
      } ]
   } ]
}
